src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<? extends Machine> get() {
+        Bulldozer bulldozer1 = new Bulldozer();
+        Bulldozer bulldozer2 = new Bulldozer();
+        Bulldozer bulldozer3 = new Bulldozer();
+        List<Bulldozer> bulldozerList = new ArrayList<>();
+        bulldozerList.add(bulldozer1);
+        bulldozerList.add(bulldozer2);
+        bulldozerList.add(bulldozer3);
+        return bulldozerList;

```

Comment:
you could do `bulldozerList.add(new Bulldozer())`
or even all of it in one line : `return List.of(new Bulldozer()...`